README.md

```diff
@@ -136,7 +136,7 @@ module.exports {
   // do not end lines of generated code with ';' if true
   withoutSemi: false,
   // class to use for repeated properties
-  repeatedClass: 'qx.data.Array',
```

Comment:
Also a change I am not ok with, this removes the opportunity to add change listeners to the property that are provided by qx.data.Array but not by the native Array. And that is the reason why qx.data.Array is used here in the first place.

Reply:
Ya, this PR is directly from my master branch so it's picking up changes I didn't intend to have in the PR. Let's just close this PR for now and I'll circle back around when I have things adjusted.